### PR TITLE
Add cancel support for topups

### PR DESCRIPTION
**Pull request by foresthu-stripe**
Add support for the new cancel topup endpoint

r? @remi-stripe || @ob-stripe 
cc @stripe/api-libraries @miguel-stripe @kenneth-stripe @kimberly-stripe

### CODE REVIEW

File: lib/resources/Topups.js [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
 var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
 
 module.exports = StripeResource.extend({
   path: 'topups',
   includeBasic: ['create', 'list', 'retrieve', 'update', 'setMetadata', 'getMetadata'],
+
+  cancel: stripeMethod({
+    method: 'POST',
+    path: '{topupId}/cancel',
```
Comment (remi-stripe) [inline lib/resources/Topups.js]: The test seem to confirm it works but other files have a leading `/` in the path.

**Comment by remi-stripe** [comment]
Added a minor comment though it seems to work. Re-assigning to @brandur-stripe to double check and then release!

File: lib/resources/Topups.js (hunk 1 above)
Comment (brandur-stripe) [reply lib/resources/Topups.js]: Hm, we seem to be a little inconsistent on this right now — looking through other examples in the library, it seems that both work.

**Comment by brandur-stripe** [comment]
LGTM.

**Comment by brandur-stripe** [comment]
Released as 6.7.0.

**Pull request merged**